server/models/offices.js

```diff
@@ -0,0 +1,25 @@
+const db = [
+  {
```

Comment:
Why is this called db? What do these objects represent?

Reply:
They have no bearing on the current project please. I was using it as a template from a previous project. I have moved to resolve it.